doc/tutorials/distributed.rst

```diff
@@ -109,17 +43,3 @@ If you are using Azure Confidential Computing, all your VMs must be on the same
 
       secure-xgboost/host/dmlc-core/tracker/dmlc-submit --cluster local --num-workers <num_nodes_in_cluster> --worker-memory 1g python3 distr-training.py
 
-  
-
-***************
```

Comment:
There is a similar troubleshooting section in the multi-client tutorial too. Can you remove that as well?

Reply:
Done